src/solvers/dgsem_tree/indicators.jl

```diff
@@ -93,16 +93,74 @@ end
 function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorHennemannGassner)
     @nospecialize indicator # reduce precompilation time
```

Comment:
```suggestion
function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorHennemannGassner)
    @nospecialize indicator # reduce precompilation time
            setup = [
            "indicator variable" => indicator.variable,
            "max. α" => indicator.alpha_max,
            "min. α" => indicator.alpha_min,
            "smooth α" => (indicator.alpha_smooth ? "yes" : "no"),
        ]
        summary_box(io, "IndicatorHennemannGassner", setup)
    end
end
```
@svengoldberg @andrewwinters5000 This should:
* fix your Julia syntax errors
* restore the `show` capabilities for the HG indicator
* make JuliaFormatter not throw up anymore

Reply:
This worked when one of the `end` statements was removed.